source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -357,7 +357,7 @@ TEST_F(NiDCPowerDriverApiTest, SetAttributeViReal64_GetAttributeViReal64ReturnsS
   set_int32_attribute(
       channel_name,
       dcpower::NiDCPowerAttributes::NIDCPOWER_ATTRIBUTE_MEASURE_WHEN,
-      dcpower::MeasureWhen::MEASURE_WHEN_NIDCPOWER_VAL_AUTOMATICALLY_AFTER_SOURCE_COMPLETE);
+      dcpower::NiDCPowerInt32AttributeValues::NIDCPOWER_INT32_VAL_MEASURE_WHEN_AUTOMATICALLY_AFTER_SOURCE_COMPLETE);
```

Comment:
I feel it would be nice if the value was same as what is in C header which is "NIDCPOWER_VAL_AUTOMATICALLY_AFTER_SOURCE_COMPLETE" . But currently it had INT32 as well as attribute name. Was there any reason to have both of those?



NIDCPOWER_VAL_AUTOMATICALLY_AFTER_SOURCE_COMPLETE
--